packages/front-end/tests/e2e/lab-technician/Labs_Lab_Technician.spec.e2e.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from 'playwright/test';
+
+test('01 - Create a Laboratory Unsuccessfully', async ({ page, baseURL }) => {
+  await page.goto(`${baseURL}/labs`);
+  await page.waitForTimeout(2000);
+
+  // confirm button if enabled/visible
+  await expect(page.getByRole('button', { name: 'Create a new Lab' })).toBeHidden();
+});
+
+test('02 - Remove a Laboratory Unsuccessfully', async ({ page, baseURL }) => {
+  await page.goto(`${baseURL}/labs`);
+  await page.waitForLoadState('networkidle');
+
+  // Check if any test lab exists
+  let hasTestLab = true;
+  try {
+    hasTestLab = await page.getByRole('row').nth(1).isVisible();
+  } catch (error) {
+    console.log('No lab not found:', error);
```

Comment:
console.log -> console.error

Message would be clearer as "No labs found"

Reply:
updated to console.error('No labs found');